MessengerOnFileForLinux/ChatStarter/src/ChatRequest.cpp

```diff
@@ -90,17 +88,17 @@ std::unique_ptr<std::string> ChatRequest::getUserStatus(const std::string& usern
 }
 
 
-bool ChatRequest::isUserActive(const User& user) const
+bool ChatRequest::isUserActive(const std::string& username) const
 {
     log_.function("ChatRequest::isUserActive() started");
 
-    std::unique_ptr<std::string> userStatusToCompare = getUserStatus(user.getUsername());
+    std::unique_ptr<std::string> userStatusToCompare = getUserStatus(username);
 
-    UserInactivityDetector userInactivityDetector(user.getUsername()); //TODO mawoznia temportary solution
+    UserInactivityDetector userInactivityDetector(username); //TODO mawoznia temportary solution
```

Comment:
Z czystej ciekawości co oznacza "temportary solution"?

Reply:
Masz tam opisane niżej, UserInactivity bedzie memberem klasy. Nie rozkopywałem tego, bo i tak cala klasa jest do refaktoru i jest na to przygotowany task